web/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_BACKEND_PORT=5000
```

Comment:
Is this still needed with the local dev proxy? Also does not match with the docs :)

Reply:
Yeah, I just checked, that is something that was left over. The way it is documented is correct here, we need a BACKEND_PORT and a BACKEND_HOST variable that are used to configure the proxy.

Reply:
I removed the whole file, I'd recommend using a `.env.development.local` file anyway, as documented.